examples/asr/speech_to_text_bpe.py

```diff
@@ -59,13 +59,19 @@ def main(cfg):
     logging.info(f'Hydra config: {cfg.pretty()}')
     trainer = pl.Trainer(**cfg.trainer)
     exp_manager(trainer, cfg.get("exp_manager", None))
+
     asr_model = EncDecCTCModelBPE(cfg=cfg.model, trainer=trainer)
 
     trainer.fit(asr_model)
 
     if hasattr(cfg.model, 'test_ds') and cfg.model.test_ds.manifest_filepath is not None:
         gpu = 1 if cfg.trainer.gpus != 0 else 0
-        trainer = pl.Trainer(gpus=gpu)
+        trainer = pl.Trainer(
+            gpus=gpu,
+            precision=cfg.trainer.precision,
+            amp_level=cfg.trainer.amp_level,
+            amp_backend=cfg.trainer.amp_backend,
```

Comment:
cfg.trainer.amp_backend is not always specified, so youll need `cfg.trainer.get('amp_backend', 'native')`. Same for `amp_level` and `precision`.

Reply:
Added.